examples/tac_deploy/tac-deployment.yaml

```diff
@@ -18,7 +18,7 @@ spec:
         kubernetes.io/os: linux
       containers:
       - name: tac-deploy-container
-        image: gcr.io/fetch-ai-sandbox/tac_deploy:0.0.13
+        image: gcr.io/fetch-ai-sandbox/tac_deploy:0.0.14
```

Comment:
why `0.0.14` and not `0.14.0`?

Reply:
No real meaning, just iterating through atm. We can adopt minor bunches next.